_scalingo_api/events.md

```diff
@@ -1771,6 +1771,99 @@ Example object:
   }
 }
 ```
+--- row ---
+
+* **Database Maintenance Planned**
+
+_When:_ A maintenance is scheduled on your database
+`type=plan_database_maintenance`
+
+{:.table}
+| field                         | type      | description                     |
+| ------------------------------| --------- | ------------------------------- |
+| maintenance_id                | string    | The related maintenance ID      |
+| addon_name                    | string    | The related addon name          |
```

Comment:
Since the [last update of the API](https://github.com/Scalingo/api/pull/2248) we want to add all available information,
i.e. adding:
- `maintenance_type`
- `maintenance_duration`
- `maintenance_duration`